src/main/webapp/app/iris/overview/base-chatbot/chat-history-item/chat-history-item.component.html

```
@@ -0,0 +1,6 @@
+<div class="chat-history-item" [ngClass]="{ 'chat-history-item-selected': active() }" (click)="onItemClick()">
```

HTMLHint (1.5.0)

[error] Doctype must be declared before any non-comment content.

(doctype-first)